src/fix_hmc.cpp

```diff
@@ -171,12 +171,12 @@ void FixHMC::post_constructor()
 /* ---------------------------------------------------------------------- */
 
 template <typename T>
-void store_peratom_member(LAMMPS_NS::Atom::PerAtom &stored_peratom_member,
+void FixHMC::store_peratom_member(LAMMPS_NS::Atom::PerAtom &stored_peratom_member,
                           LAMMPS_NS::Atom::PerAtom current_peratom_member, int nlocal)
 {
   if (stored_peratom_member.name.compare(current_peratom_member.name)) {
     printf(
-        "NONONONONONO\n");    //error->all(FLERR, "fix hmc tried to store incorrect peratom data");
+        error->all(FLERR, "fix hmc tried to store incorrect peratom data");
```

Comment:
Looks much more professional, but the error statement could be more informative regarding how the user can fix it
```suggestion
        error->all(FLERR, "fix hmc tried to store incorrect peratom data. Ensure the input atoms match those used in fix hmc");
```